scripts/status_consistency.py

```diff
@@ -0,0 +1,36 @@
+from app import geo
```

Comment:
nit: it'd be good to have a description of what this script does, even if it's brief (e.g. it was difficult to figure out what the scripts did post-Kona)